cmd/config.go

```diff
@@ -217,21 +216,6 @@ func getSharedIterationsExecution(iterations null.Int, duration types.NullDurati
 	return scheduler.ConfigMap{lib.DefaultSchedulerName: ds}
 }
 
-func checkExecutionMismatch(observedExecution, derivedExecution scheduler.ConfigMap, shortcutField string) error {
-	if observedExecution == nil {
-		return nil
-	}
-
-	// Work around the v0.24.0 archive metadata.js options that wrongly contain execution
-	if !reflect.DeepEqual(observedExecution, derivedExecution) {
-		errMsg := fmt.Sprintf("specifying both `%s` and `execution` is not supported", shortcutField)
-		return executionConflictConfigError(errMsg)
-	}
-
-	log.Warnf("ignoring the specified `execution` options because they match the ones derived from `%s`", shortcutField)
```

Comment:
This is the warning that is expected in https://github.com/loadimpact/k6/pull/1076/files#diff-2a79d104987fee66e270d22f7cee6ddeL406

Reply:
:+1: